gui/src/system/SystemPages.tsx

```diff
@@ -374,19 +386,31 @@ async function submitConfig(edited: boolean) {
       null : JSON.parse(mappingvalvenormcloseds[i].value.toLowerCase())
   }
   console.log(entries);
-  const success = await sendConfig(serverIp() as string, {id: configName, mappings: entries} as Config) as object;
+  const response = await sendConfig(serverIp() as string, {id: configName, mappings: entries} as Config) as Response;
+  const success = response as object;
```

Comment:
Why does this `success` object even exist? Can't we just index on `response`? Also, this casting isn't used completely properly, since `success` is initialized before we've confirmed that the request was successful.